test/integration/master/synthetic_master_test.go

```diff
@@ -225,6 +225,72 @@ func TestStatus(t *testing.T) {
 	}
 }
 
+var expected1 = `etcdserver: request is too large`
+var expected2 = `rpc error: code = ResourceExhausted desc = grpc: trying to send message larger than max (3489913 vs. 2097152)`
```

Comment:
would at least remove the concrete numbers from the string. They might change.

Reply:
@sttts Yeah right, will remove that